lib/CLI/Types.hs

```diff
@@ -56,6 +56,7 @@ data CopyResult
   | CPRPathNotFound Path
   | CPRMissingEntryName
   | CPRDestinationIsDirectory [(EntryPath, EntryPath)]
+  | CPRDestinationHasEntry EntryPath
```

Comment:
`CPRDestinationIsDirectory` and `CPREntryAlreadyExists` store a list with all the problematic entries/paths.

I think `CPRDestinationHasEntry` should do the same.

Reply:
Done